ckanext/spectrum/templates/header.html

```diff
@@ -1,35 +1,124 @@
 {% ckan_extends %}
```

Comment:
Sometimes, there are html tags that are not in small blocks but big blocks,

Reply:
I have reverted the header.html the way it is currently in master branch. I created a new css file named navbar-expand.css which overrides the default .navbar-expand-lg of bootstrap 5. Since it is hard to update the html file without whether changing the ckan core template or having a big chunk of code in the ckanext-spectrum header.html file. I have verified, this class is used once only in Project in spectrum-ckan/ckan/ckan/templates/header.html